### ISSUE TITLE
Capitalized command bug

### ISSUE THREAD

**Issue by HotPotatoC** [HotPotatoC/kvstore#2]
When using a capitalized command like `GET` instead of `get` it returns `<nil>`

```sh
127.0.0.1:7275> set person john
0.000293s


127.0.0.1:7275> get person
0.000353s
john

127.0.0.1:7275> GET person
0.000326s
<nil>```

**Comment by HotPotatoC** [comment]
how tf do you fix this?

**Comment by HotPotatoC** [comment]
fixed! b0c1192
